engine/profiler/src/profiler.cpp

```diff
@@ -594,7 +596,9 @@ static void SampleTreeCallback(void* _ctx, const char* thread_name, dmProfile::H
 
 static void ProcessProperty(dmProfileRender::ProfilerFrame* frame, int indent, dmProfile::HProperty property)
 {
-    uint32_t name_hash = dmProfile::PropertyGetNameHash(property);
+    const char* name = dmProfile::PropertyGetName(property);
+    uint32_t name_hash = dmHashString32(name?name:"<empty_property_name>");
+
```

Comment:
Now we need to hash this every frame instead.

Reply:
;(